Makefile

```diff
@@ -184,7 +184,7 @@ $(BLD_DIR_ENV)/stamp:
 ifeq ($(PYTHON_VER),python3.9)
 	@echo "--- Creating virtual environment at $(BLD_DIR_ENV) using $(PYTHON_VER)"
 	@$(SYS_PYTHON) -m pip install --upgrade pip==22.2.2
-	@$(SYS_PIP) install virtualenv==$(VIRTUAL_ENV_VERSION) virtualenv-make-relocatable==$(VIRTUAL_ENV_RELOCATABLE_VERSION)
+	@$(SYS_PIP) install virtualenv==20.19.0 virtualenv-make-relocatable==0.0.1
```

Comment:
Why are we not just setting `VIRTUAL_ENV_VERSION` to `20.19.0` for the whole thing?

Reply:
virtualenv==20.24.4 works fine for python3.8. Only rhel9 uses python3.9. So I'm changing the version of virtual env here.

Reply:
I would prefer keeping a single version of virtualenv everywhere as far as possible. So, if everything works fine on `20.19.0`, then let's go with that. Also, place a comment next to where you have defined the `VIRTUAL_ENV_VERSION` about how rhel9/python3.9 does not yet work for anything beyond `20.19.0`.